applications/MeshingApplication/custom_processes/mmg_process.cpp

```diff
@@ -510,7 +517,15 @@ void MmgProcess<TMMGLibrary>::ExecuteRemeshing()
     }
 
     /* Save to file */
-    if (save_to_file) SaveSolutionToFile(true);
+    if (save_to_file) {
+        if (GetMmgVersion() == "5.5") {
+            if (mDiscretization != DiscretizationOption::ISOSURFACE) {
```

Comment:
Because the interpolation of the solution after remesh changes slightly

Reply:
And it does not happen in previous versions because now the isosurface can have a metric?

Shouln't we prevent the isosurface case from printing the solution also for versions before 5.5?

Reply:
Well, we need to check something...

Reply:
Sure, my question was why is this required in the new version but no the old one:

https://github.com/KratosMultiphysics/Kratos/blob/b7e24f9ba73befba37051ceeb3191f6a9e7de736/applications/MeshingApplication/custom_processes/mmg_process.cpp#L519-L528

Or if it was a mistake that the check is not in the old versions.

In any case it is a minor comment